src/client/components/pages/RoomSchedule/RoomSchedule.tsx

```diff
@@ -21,10 +21,19 @@ import {
 import React, {
   FunctionComponent, useCallback, useContext, useEffect, useState,
 } from 'react';
+import styled from 'styled-components';
 import { CourseListing, SessionBlock, WeekBlock } from '../Schedule/blocks';
 import DayBlock from '../Schedule/blocks/DayBlock';
 import { SemesterSelection } from '../Schedule/SchedulePage';
 
+/**
+ * A wrapper component used to format the Room Selector
+ */
+const RoomSelectorWrapper = styled.span`
+  padding-right: 25%;
```

Comment:
This is a good idea to use the Mark One theme values, but in this case, applying the changes, the layout looks like this with a lot of space between the two filters: 
<img width="1436" alt="image" src="https://user-images.githubusercontent.com/29589689/207967390-b70b3a56-a508-43bc-9b51-f8ebc84d1db9.png">

According to the [UI suggestion](https://github.com/seas-computing/course-planner/issues/584) from Vittorio, it seems like we should have the two inputs closer together. I think for now it makes sense to keep the value as a percentage of the page, because we do not have a value large enough in Mark One to achieve the spacing that we want.